### PR TITLE
Lighthouse tests with Mobify Preview

### PR DESCRIPTION
**Pull request by ellenmobify**
The previous PR (https://github.com/mobify/progressive-web-scaffold/pull/351) to re-enable the Lighthouse tests for CI did not accurately report results when testing a local build through Preview. This stemmed from the use of a proxy to allow Lighthouse to test a local build. 

The addition of https://github.com/mobify/portal_app/pull/3014 means we can now enable requests under a Preview environment without using a proxy. This means we can get accurate Lighthouse scores for local builds. Yay!

## Changes
- Use a custom user agent containing "MobifyPreview" so that Preview will accept our requests.
- Remove files and dependencies related to the proxy.

## How to test-drive this PR
- Run `npm run dev`
- In another tab, run `npm run test:pwa-local`

Or find the build in CI: https://circleci.com/gh/mobify/progressive-web-scaffold/2507. The report will be under the Artifacts tab.

### CODE REVIEW

File: web/scripts/lighthouse-ci.sh [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-		--output-path=${OUTPUT_PATH} \
-		${URL}"
+# Finally, parse the HTML report for the Lighthouse score. 
+# CI will fail the build if the score is below a threshold.
+# See min_lighthouse_score in package.json
+
+# Change www.merlinspotions.com to the project's URL. 
+
+concurrently --kill-others --success --raw \
+	'chrome-debug --allow-insecure-localhost --user-agent="Chrome MobifyPreview"' \
```
Comment (johnboxall) [inline web/scripts/lighthouse-ci.sh]: @ellenmobify Lighthouse 1.6.0 merged the ability to pass Chrome flags directly to the `lighthouse` command:

https://github.com/GoogleChrome/lighthouse/releases/tag/1.6.0

You should be able to something like `lighthouse --chromeflags='--allow-insecure-localhost ...'

Which should also get rid of the need to use `concurrently` at all here.

File: web/scripts/lighthouse-local.sh [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
-		--skip-autolaunch \
-		--output=html \
-		--output-path=${OUTPUT_PATH} \
-		${URL}"
+# Change www.merlinspotions.com to the project's URL. 
 
-open $OUTPUT_PATH
+concurrently --kill-others --success --raw \
+	'chrome-debug --allow-insecure-localhost --user-agent="Chrome MobifyPreview"' \
+	"sleep 5 && lighthouse --skip-autolaunch --output=html \
```
Comment (johnboxall) [inline web/scripts/lighthouse-local.sh]: @ellenmobify Lighthouse 1.6.0 includes the `--view` option which is a cross platform `open`.

You should be able to:
1) Remove `concurrently` by passing the chrome flags directly to LH
2) Pass the `--view` argument to Lighthouse to automatically open the report.

**Comment by johnboxall** [comment]
@ellenmobify looks good – with the Lighthouse 1.6.0 release we can actually make these scripts even less complex and drop using concurrently.

File: web/scripts/lighthouse-local.sh (hunk 2 above)
Comment (ellenmobify) [inline web/scripts/lighthouse-local.sh]: Looks like the `--view` functionality will only look under a `reports` subdirectory.

File: web/package.json [hunk 3]
```diff
@@ -13,6 +13,7 @@
   "dependencies": {
     "babel-runtime": "6.11.6",
     "es6-promise": "4.0.5",
+    "http-server": "^0.9.0",
```
Comment (jeffkamo) [inline web/package.json]: Should this be moved back down to be alphabetical?

File: web/lighthouse/cert-db-password [hunk 4]
```diff
@@ -0,0 +1 @@
+p4ssword
```
Comment (kschmidtdev) [inline web/lighthouse/cert-db-password]: Normally we would put this in an environment variable https://mobify.atlassian.net/wiki/display/SYS/Best+Practices+For+Keeping+Secrets+in+Applications

**Pull request merged**